src/docs/cookbook/design/tabs.md

```diff
@@ -6,21 +6,25 @@ Working with tabs is a common pattern in apps following the Material Design
 guidelines. Flutter includes a convenient way to create tab layouts as part of
 the [material library](https://docs.flutter.io/flutter/material/material-library.html).
 
-## Directions
+## Steps
 
-  1. Create a `TabController`
-  2. Create the tabs
-  3. Create content for each tab
+  1. Create a [`TabController`](https://docs.flutter.io/flutter/material/TabController-class.html)
+  2. Create the [`TabBar`](https://docs.flutter.io/flutter/material/TabBar-class.html)
+     with [`Tab's`](https://docs.flutter.io/flutter/material/Tab-class.html) 
+  3. Create content for each tab in a [`TabBarView`](https://docs.flutter.io/flutter/material/TabBarView-class.html)
 
 ## 1. Create a `TabController`
 
-In order for tabs to work, we'll need to keep the selected tab and content
-sections in sync. This is the job of the [`TabController`](https://docs.flutter.io/flutter/material/TabController-class.html).
+The purpose of the [`TabController`](https://docs.flutter.io/flutter/material/TabController-class.html),
+is to coordinate the selected `Tab` in the `TabBar` and content in `TabBarView`. 
 
-We can either manually create a `TabController` or use the
-[`DefaultTabController`](https://docs.flutter.io/flutter/material/DefaultTabController-class.html)
-Widget. Using the `DefaultTabController` is the simplest option, since it will
-create a `TabController` for us and make it available to all descendant Widgets.
+A `TabController` can be either:
```

Comment:
I like the idea of this section, but the language is a bit awkward. Maybe:

 > A `TabController` can be:
 >
 >  - Created manually 
 >  - Created automatically by using a `DefaultTabController` Widget
 >
 > Using the `DefaultTabController` is the simplest...